packages/web3-utils/test/fixtures/converters.ts

```diff
@@ -0,0 +1,241 @@
+	['-123', 'Invalid value given "-123". Error: not valid hex string.'],
+	['-9x123', 'Invalid value given "-9x123". Error: not valid hex string.'],
+];
+
+export const utf8ToHexValidData: [string, HexString][] = [
+	['I have 100£', '0x49206861766520313030c2a3'],
+	['I \u1234data', '0x4920e188b464617461'],
+	['I ♥ data', '0x4920e299a52064617461'],
+	['I \u0000 data', '0x4920002064617461'],
+	['\u0000 null suffix', '0x206e756c6c20737566666978'],
+	['null prefix\u0000', '0x6e756c6c20707265666978'],
+	['\u0000', '0x'],
+];
+
+export const utf8ToHexInvalidData: [any, string][] = [
+	[12, 'Invalid value given "12". Error: not a valid string.'],
+	[BigInt(12), 'Invalid value given "12". Error: not a valid string.'],
+	[null, 'Invalid value given "null". Error: not a valid string.'],
+	[undefined, 'Invalid value given "undefined". Error: not a valid string.'],
+	[{}, 'Invalid value given "[object Object]". Error: not a valid string.'],
+	[true, 'Invalid value given "true". Error: not a valid string.'],
+	[false, 'Invalid value given "false". Error: not a valid string.'],
+];
+
+export const hexToUtf8ValidData: [HexString, string][] = [
+	['0x49206861766520313030c2a3', 'I have 100£'],
+	['0x4920e188b464617461', 'I \u1234data'],
+	['0x4920e299a52064617461', 'I ♥ data'],
+	['0x4920002064617461', 'I \u0000 data'],
+	['0x206e756c6c20737566666978', ' null suffix'],
+	['0x6e756c6c20707265666978', 'null prefix'],
+	['0x', ''],
+];
+
+export const hexToUtf8InvalidData: [any, string][] = [
+	[
+		'0x4920686176652031303g0c2a3',
+		'Invalid value given "0x4920686176652031303g0c2a3". Error: not valid hex string.',
+	],
+	['afde', 'Invalid value given "afde". Error: not valid hex string.'],
+	[null, 'Invalid value given "null". Error: can not parse as byte data.'],
+	[undefined, 'Invalid value given "undefined". Error: can not parse as byte data.'],
+	[{}, 'Invalid value given "[object Object]". Error: can not parse as byte data.'],
+	[true, 'Invalid value given "true". Error: can not parse as byte data.'],
+];
+
+export const asciiToHexValidData: [string, HexString][] = [
+	['I have 100', '0x49206861766520313030'],
+	['I \u1234data', '0x49203464617461'],
+	['I data', '0x492064617461'],
+	['I \u0000 data', '0x4920002064617461'],
+	['\u0000 null suffix', '0x00206e756c6c20737566666978'],
+	['null prefix\u0000', '0x6e756c6c2070726566697800'],
+	['\u0000', '0x00'],
+	['', '0x'],
+];
+
+export const hexToAsciiValidData: [string, HexString][] = [
+	['0x49206861766520313030', 'I have 100'],
+	['0x49203464617461', 'I 4data'],
+	['0x492064617461', 'I data'],
+	['0x4920002064617461', 'I \u0000 data'],
+	['0x00206e756c6c20737566666978', '\u0000 null suffix'],
+	['0x6e756c6c2070726566697800', 'null prefix\u0000'],
+	['0x00', '\u0000'],
+	['0x', ''],
+];
+
+export const toHexValidData: [Numbers | Bytes | Address | boolean, [HexString, ValueTypes]][] = [
+	[1, ['0x1', 'uint256']],
+	[255, ['0xff', 'uint256']],
+	[256, ['0x100', 'uint256']],
+	[BigInt(12), ['0xc', 'bigint']],
```

Comment:
```suggestion
	[BigInt(12), ['0xc', 'bigint']],
	[12n, ['0xc', 'bigint']],
```

Add test case for alternative syntax for BigInt